src/components/app/create/CreateApp.tsx

```diff
@@ -281,197 +233,178 @@ export class AddNewApp extends Component<AddNewAppProps, AddNewAppState> {
-                    headerClassName={'m-20'}
-                    close={this.props.close}
-                    onSave={(e) => {
-                        e.preventDefault();
-                        this.createApp();
-                    }}
-                    closeOnESC={true}
-                >
-                    <hr className="separator m-0" />
-                    <div className="scrollable-content p-20">
-                        <label className="form__row">
-                            <span className="form__label">App Name*</span>
-                            <input
-                                ref={(node) => (this._inputAppName = node)}
-                                className="form__input"
-                                type="text"
-                                name="app-name"
-                                value={this.state.form.appName}
-                                placeholder="e.g. my-first-app"
-                                autoComplete="off"
-                                autoFocus={true}
-                                tabIndex={1}
-                                onChange={this.handleAppname}
-                                required
-                            />
-                            <span className="form__error">
-                                {appNameErrors && !this.state.isValid.appName ? (
-                                    <>
-                                        <Error className="form__icon form__icon--error" />
-                                        {errorObject[0].message} <br />
-                                    </>
-                                ) : null}
-                            </span>
-                            <span className="form__text-field-info form__text-field-info--create-app">
-                                <Info className="form__icon form__icon--info form__icon--create-app" />
-                                Apps are NOT env specific and can be used to deploy to multiple environments.
-                            </span>
-                        </label>
-                        <div className="form__row">
-                            <span className="form__label">Project*</span>
-                            <ReactSelect
-                                className="m-0"
-                                tabIndex={2}
-                                isMulti={false}
-                                isClearable={false}
-                                options={this.state.projects}
-                                getOptionLabel={(option) => `${option.name}`}
-                                getOptionValue={(option) => `${option.id}`}
+        ]
+        let showError = this.state.showErrors
+        let appNameErrors = this.state.appNameErrors
+        return (
+            <div className="scrollable-content p-20">
+                <div className="form__row">
+                    <span className="form__label">App Name*</span>
```

Comment:
Just From UI POV: Could we make * red as it is a mandatory input

Reply:
Will do in next PR